src/components/esrijs/MapView.js

```diff
@@ -41,6 +42,10 @@ const ReactMapView = ({ setView, zoomToGraphic, onClick }) => {
       },
     });
 
+    if (window.Cypress) {
```

Comment:
**thought**: this seems a little leaky. is it necessary?

Reply:
Leaky? Not sure. I was just trying to prevent this from being run in production. Maybe we just run it all of the time? 

This code along with the new module for writing to window is something that we should definitely roll up into some reuseable. For now, it's just an experiment to see what the best pattern will be.

Reply:
yeah, I think the real definition is that the abstraction leaks things out that it's supposed to abstract away. I'm using it in the sense that the testing code is leaking to places outside of the testing code. This boils down to my preference to not modify our components to be successful with cypress tests. Maybe that isn't an option for using cypress?

Reply:
I see what you are meaning now. Test code leaking into our source code. Ya, I don't like it either. I'm definitely open to other ideas. Basically, this enables cypress to make assertions on the state of the map via the window object. I'd love to come up with something better in the future. This is intended to just start the experimentation process.